src/unix/mod.rs

```diff
@@ -328,6 +328,10 @@ extern {
     #[cfg_attr(all(target_os = "macos", target_arch = "x86"),
                link_name = "fdopen$UNIX2003")]
     pub fn fdopen(fd: ::c_int, mode: *const c_char) -> *mut ::FILE;
+
+    #[cfg(not(any(target_os = "macos", target_os = "ios")))]
```

Comment:
Ah yeah let's just inline this definition in modules other than `src/unix/bsd/apple`